src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -462,7 +462,7 @@ PharoShortcuts >> selectBackwardLineShortcut [
 
 { #category : #'keymaps - editor selection' }
 PharoShortcuts >> selectBackwardWordShortcut [
-	^ Character arrowLeft meta shift
+	^ Character arrowLeft shift ctrl
```

Comment:
I'm not sure that this change will work on Mac.

Reply:
Changed + Added comment